jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/PortForwardServiceTest.java

```diff
@@ -92,14 +81,30 @@ public void testSimpleScenario() throws Exception {
         OpenShiftClient client = mockServer.getOpenshiftClient();
         PortForwardService service = new PortForwardService(client, logger) {
             @Override
-            public ProcessUtil.ProcessExecutionContext forwardPortAsync(KitLogger externalProcessLogger, String pod, String namespace, int remotePort, int localPort) throws JKubeServiceException {
-                return new ProcessUtil.ProcessExecutionContext(process, Collections.<Thread>emptyList(), logger);
+            public LocalPortForward forwardPortAsync(String podName, String namespace, int remotePort, int localPort) {
+                return client.pods().inNamespace(namespace).withName(podName).portForward(localPort, remotePort);
             }
         };
 
-        try (Closeable c = service.forwardPortAsync(logger, new LabelSelectorBuilder().withMatchLabels(Collections.singletonMap("mykey", "myvalue")).build(), 8080, 9000)) {
+        try (Closeable c = service.forwardPortAsync(new LabelSelectorBuilder().withMatchLabels(Collections.singletonMap("mykey", "myvalue")).build(), 8080, 9000)) {
             Thread.sleep(3000);
         }
     }
 
+    @Test
+    public void startPortForward(
+        @Mocked KubernetesClient kubernetesClient, @Mocked KitLogger logger,
+        @Mocked LocalPortForward lpf, @Mocked PortForwardTask pft
```

Comment:
Variable is not used, but mock is needed (whitebox). I'll probably add a comment to reflect that since someone might be tempted to remove the variable.

Reply:
Umm, strange. When I deleted this variable and re ran the test it was passing. I'll probably check again

Reply:
The test might pass (now), but it's invoking functionality from a different unit (LocalPortForward).
This means 2 things:
 - If you change that class in the future, the test might stop passing.
 - The test is slower because it's running+testing code from a different unit.